src/main/java/com/dougnoel/sentinel/steps/ImageVerificationSteps.java

```diff
@@ -54,48 +59,78 @@ public static void verifyImageNotMatch(String elementName, String assertion) thr
         
         //Check the result after determining if we're doing a should match or should not match.
 		if(negate) {
-			assertFalse(expectedResult, comparisonResult.getImageComparisonState().equals(ImageComparisonState.MATCH));
+			assertNotEquals(expectedResult, comparisonResult.getImageComparisonState(), ImageComparisonState.MATCH);
 		}
 		else {
-			assertTrue(expectedResult, comparisonResult.getImageComparisonState().equals(ImageComparisonState.MATCH));
+			assertEquals(expectedResult, comparisonResult.getImageComparisonState(), ImageComparisonState.MATCH);
 		}
 	}
 	
 	/**
-	 * Takes an updated screenshot of the current element, or page, for comparison to the earlier expected image.
-	 * Will enlarge both images to be the same size using the retrieved background color for
-	 * the new empty space if their sizes are not equal.
-	 * Then runs, and returns, an ImageComparisonResult of the two images.
-	 * 
+	 * Takes an updated screenshot of the current element, or page, for comparison to an earlier expected image.
+	 * <br><br>
+	 * <i>
+	 *     If screenshots are different sizes, both will be enlarged to the largest dimensions found. Expanded space
+	 * will be set to parent background color for web, upper-left-most color for windows applications.
+	 * </i>
+	 *
 	 * @param elementName String the name of the element to capture and compare 
 	 * or any casing of "page" alone to compare the entire page.
 	 * @throws IOException if file creation does not work
 	 * 
 	 * @return ImageComparisonResult the image comparison result.
 	 */
     private static ImageComparisonResult compareImages(String elementName) throws IOException {
```

Comment:
See additional comments below, but this method breaks encapsulation. It should exist inside of the screenshots themselves. If there are special cases to handle them you should wrap the screenshots and handle that.